backend/apis/clients/follow-ups/patch-follow-up.api.js

```diff
@@ -78,6 +81,9 @@ app.patch("/api/clients/:clientId/follow-ups/:followUpId", (req, res) => {
         [id, newFollowUp.id]
       );
     });
+
+    // update client logs here as well???? TBD
```

Comment:
we should insert a new log, and then update the old logs `idOfUpdatedLog` column.